### PR TITLE
Add manual test for color space rendering behavior

### PR DESCRIPTION
**Pull request by mattsarett**
The following content should render with noticeable differences depending on whether Flutter is using color correct Skia.

### CODE REVIEW

File: examples/flutter_gallery/lib/demo/color_testing_demo.dart
Comment (mattsarett) [inline examples/flutter_gallery/lib/demo/color_testing_demo.dart]: These are placeholders for the actual images that I want to use.  Can you instruct on how to add images to flutter_gallery_assets?

**Comment by mattsarett** [comment]
@chinmaygarde

**Comment by mattsarett** [comment]
https://github.com/flutter/flutter/issues/8152

**Comment by HansMuller** [comment]
This manual test should really be a standalone app in dev/manual_tests, not a page in the gallery demo.

**Review by HansMuller — DISMISSED** [review]
I left a comment already about moving the code to our manual tests directory; adding this comment just in case you were waiting for a review signal from github

**Comment by mattsarett** [comment]
Got it thanks, I'll move this to a test app.

**Comment by mattsarett** [comment]
@HansMuller Are you able to explain how to run the files in the dev/manual_tests?

**Comment by abarth** [comment]
```
cd dev/manual_tests
flutter run page_view.dart
```

**Comment by mattsarett** [comment]
I'm running in to an error:

msarett@msarett1:~/flutter/flutter/dev/manual_tests$ ../../bin/flutter run page_view.dart 
Launching page_view.dart on Pixel XL in debug mode...
The build process for Android has changed, and the current project configuration
is no longer valid. Please consult

  https://github.com/flutter/flutter/wiki/Upgrading-Flutter-projects-to-build-with-gradle

for details on how to upgrade the project.

**Comment by reed-at-google** [comment]
very exciting.

**Comment by eseidelGoogle** [comment]
@mattsarett do you have more than one copy of flutter installed?  That message is very old, and would suggest that you have a newer copy of `flutter` in your path than the repository you're working from.  I say that, because the example you're working from is flutter_gallery, which certainly should have been upgraded already to the new android build setup? :)

**Comment by mattsarett** [comment]
@eseidelGoogle I'm no longer working in Flutter gallery because I'm hoping to move my test code into dev/manual_tests.  Would that make a difference?  I'll go ahead and see if I seem to be out of date anywhere.

**Comment by eseidelGoogle** [comment]
Ah, dev/manual_tests is probably out of date, yes.  @jason-simmons or @yjbanov could probably help us upgrade that for you.

**Comment by mattsarett** [comment]
Ah got it thanks, I appreciate the help!

**Comment by yjbanov** [comment]
FYI: https://github.com/flutter/flutter/pull/10057 is fixing the upgrade issue

**Comment by mattsarett** [comment]
Thanks Jason and Yegor!  This should be ready after #10057 lands.

**Comment by mattsarett** [comment]
@HansMuller please take a look.  I've moved the test into dev/manual_tests.

**Comment by Hixie** [comment]
We shouldn't check in graphics assets into this repo. Can you use the flutter_gallery_assets package instead? See `dev/benchmarks/complex_layout` for an example.

**Comment by mattsarett** [comment]
Thanks for the example.  Looks like I'll want to land a change to the flutter_gallery_assets repo first?

**Comment by mattsarett** [comment]
@Hixie  Is it ok to push directly to that repo or is there a review system?

**Comment by Hixie** [comment]
I'm afraid you have entered a dark place. :-)

You want to submit a review on https://flutter-review.googlesource.com/ (the repo itself is https://flutter.googlesource.com/gallery-assets/). You'll want to make sure you describe the license for each file in the README. 

To actually submit the code review you'll want to follow the esoteric steps described here:
https://gerrit-review.googlesource.com/Documentation/user-upload.html

I am really sorry about the complexity here. If this all seems like far too much red tape to contribute to a project then please don't hesitate to just leave this PR open and eventually we'll deal with the images ourselves. (In that case, please do leave a comment here describing the license situation for each image and also maybe the technical details of each image file, for example what is important about each image such as what color spaces each image must use and what colors it must include and so forth, so that if we "optimize" them by converting to JPEG or using `optipng` or something we don't break everything.)

**Comment by mattsarett** [comment]
No problem, thanks for the detail.  I'll follow up if I have questions.

**Comment by Hixie** [comment]
Ok the images are now available at:

https://flutter.github.io/assets-for-api-docs/tests/colors/gbr.png
https://flutter.github.io/assets-for-api-docs/tests/colors/tf.png
https://flutter.github.io/assets-for-api-docs/tests/colors/wide-gamut.png

I recommend changing the app to use [Image.network] and just fetch them by URL, rather than trying to put them into a package.

**Comment by mattsarett** [comment]
Thanks very much Ian!  I think the latest version follows your suggestion.  I'll need to verify the behavior when I'm back in the office tomorrow.

**Comment by Hixie** [comment]
This needs a test (next to https://github.com/flutter/flutter/blob/master/dev/manual_tests/test/card_collection_test.dart), but other than that LGTM.

**Comment by mattsarett** [comment]
Got it.  I've added a very basic test.

**Comment by Hixie** [comment]
LGTM. Will land when build is green.

**Pull request merged**
